### ISSUE TITLE
Name of days in columns does not match the date

### ISSUE THREAD

**Issue by Giriton** [fdreyfs/vaadin-tuning-datefield#5]
Hello,

this is great addon, however we have some issues.

After we call .setLocale( ... ) on instance of InlineTuningDateField, the ENG names of months and days correctly changes to our (Czech) ones.

However in Czech locale, week starts by monday ("Po" - "Pondělí" in czech) but InlineTuningDateField still keeps showing Sunday as the first day of the week, including gray color in first column.

So we called method .setFirstDayOfWeek(1) on InlineTuningDateField , which caused gray color to be present correctly in 6th and 7th column, however names of the days do not match now! As you can see on the pic in attachment, InlineTuningDateField draws for example 6.12.2014 (saturday) correctly in 6th column, correctly by gray color, but marks it incorrectly as friday ("Pá" - "Pátek" in czech). 

Basically, names of all the days are moved by one position.

![giriton](https://cloud.githubusercontent.com/assets/10220249/5470655/0b43cb98-85eb-11e4-8a6d-731d8656b15a.png)
